network/p2p/peerstore/peerstore.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2019-2023 Algorand, Inc.
+// This file is part of go-algorand
+//
+// go-algorand is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// go-algorand is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with go-algorand.  If not, see <https://www.gnu.org/licenses/>.
+
+package peerstore
+
+import (
+	"context"
+	"fmt"
+
+	ds "github.com/ipfs/go-datastore"
+	pebbledb "github.com/ipfs/go-ds-pebble"
```

Comment:
`go-ds-pebble` [pulls in the ipfs logger](https://github.com/ipfs/go-ds-pebble/blob/master/datastore.go#L16). Is this a problem?

Reply:
Answering my own question. This is a wrapper around Uber's zap and we already have some references in the go.sum:

<img width="760" alt="image" src="https://github.com/algorand/go-algorand/assets/291133/0788d8f0-fbbb-443c-9d7b-8ae8465c5ee2">